File: CS/WindowsApplication3/MVVM/Views/OrdersTestView.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Windows.Forms;
using DevExpress.XtraBars.Docking2010;

namespace MVVMSample {
    public partial class OrdersView : UserControl {
        public OrdersView() {
            InitializeComponent();
            if (!mvvmContext.IsDesignMode) {
                InitializeBindings();
            }
        }
        private void InitializeBindings() {
            var fluent = mvvmContext.OfType<OrdersViewModel>();
            fluent.SetBinding(treeList1, c => c.DataSource, x => x.Entities);
            fluent.BindCommand(windowsUIButtonPanel1.Buttons[0] as WindowsUIButton, x => x.Clear());
            fluent.BindCommand(windowsUIButtonPanel1.Buttons[1] as WindowsUIButton, x => x.RemoveLast());
            fluent.BindCommand(windowsUIButtonPanel1.Buttons[2] as WindowsUIButton, x => x.Add());
        }
    }
}

File: VB/WindowsApplication3/MVVM/Views/OrdersTestView.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Windows.Forms
Imports DevExpress.XtraBars.Docking2010

Namespace MVVMSample
    Partial Public Class OrdersView
        Inherits UserControl

        Public Sub New()
            InitializeComponent()
            If Not mvvmContext.IsDesignMode Then
                InitializeBindings()
            End If
        End Sub
        Private Sub InitializeBindings()
            Dim fluent = mvvmContext.OfType(Of OrdersViewModel)()
            fluent.SetBinding(treeList1, Function(c) c.DataSource, Function(x) x.Entities)
            fluent.BindCommand(TryCast(windowsUIButtonPanel1.Buttons(0), WindowsUIButton), Sub(x) x.Clear())
            fluent.BindCommand(TryCast(windowsUIButtonPanel1.Buttons(1), WindowsUIButton), Sub(x) x.RemoveLast())
            fluent.BindCommand(TryCast(windowsUIButtonPanel1.Buttons(2), WindowsUIButton), Sub(x) x.Add())
        End Sub
    End Class
End Namespace
